bin/test

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+cookiecutter \
+  --no-input \
+  --config-file bin/test-config.yml \
+  .
+
+cd ./TestProject
+
+bin/test
```

Comment:
this was at first confusing ... would it be worth renaming this file (not the on in the test project)?